### DISCUSSION TITLE
83E69BB1-431A-71DE-AD08-3232A52814FA

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#12] [General]
# Pōwehi

This addon adds the central black hole of the galaxy Messier 87, also known as Powehi, which was the first black hole photographed. On April 10, 2019, the Event Horizon Telescope (EHT) released the photo.

https://celestia.mobi/addon?item=83E69BB1-431A-71DE-AD08-3232A52814FA&lang=en

**Comment by Dirgham2015** [comment]
Omg

**Comment by xuandayewudi** [comment]
😃

**Comment by dominic618** [comment]
Fun

**Comment by Alaxkibz** [comment]
legal recomendo

**Comment by curnyloveautentik-maker** [comment]
Se una de agujero negro cuando esta en descarga?????
